storage/google/cloud/storage/client.py

```diff
@@ -194,7 +194,7 @@ def lookup_bucket(self, bucket_name):
         except NotFound:
             return None
 
-    def create_bucket(self, bucket_name):
+    def create_bucket(self, bucket_name, requester_pays=None):
```

Comment:
I would say "no" to both:

- `Client.create_bucket` actually gets the thing constructed on the server side, such that the `billing/requesterPays` setting affects all subsequent API requests for the bucket.
- ` Client.bucket()` / `Bucket.__init__` allow the user to configure the bucket arbitrarily *before* the call to `Bucket.create()`.  I don't see a value in providing noisy alternatives to using the properties.